docs/source/markdown/podman-attach.1.md

```diff
@@ -9,48 +9,52 @@ podman\-attach - Attach to a running container
 **podman container attach** [*options*] *container*
 
 ## DESCRIPTION
-The attach command allows you to attach to a running container using the container's ID
-or name, either to view its ongoing output or to control it interactively.
+**podman attach** allows you to attach to a running *container* using the *container's name* or *ID*, either to view its ongoing output or to control it interactively.
```

Comment:
I realize this isn't your change, but I bet @TomSweeneyRedHat is going to jump in here and say something about not using "you"...

Reply:
indeed, 'you' is not good!
```suggestion
**podman attach** attaches to a running *container* using the *container's name* or *ID*, to either view its ongoing output or to control it interactively.
```